src/elements/Accordion/accordion.less

```diff
@@ -2,54 +2,46 @@
  * Accordion
  */
 
-@ui-accordion-background: @ui-color-turquoise;
-@ui-accordion-notification-border: 2px solid @ui-color-turquoise-dark;
-@ui-accordion-border: 2px solid @ui-color-gray-light;
-@ui-accordion-toggle-color: @ui-color-blue;
-
 .ui-accordion {
```

Comment:
Will these style changes also break backwards compatibility? Does the app need updates to adhere to these changes?